.gitignore

```diff
@@ -1,4 +0,0 @@
-_site/
-.sass-cache/
-.jekyll-metadata
-*~
```

Comment:
I think maybe you should restart from a cleaner branch. This will delete existing files in the repo .